apps/core/scenes/bookingDetail/menuGroups/Passengers/VisaDetail.js

```diff
@@ -1,67 +1,31 @@
 // @flow
 
 import * as React from 'react';
-import { View } from 'react-native';
-import { StyleSheet, Text, Icon } from '@kiwicom/universal-components';
-import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+import { VisaDetail } from '@kiwicom/margarita-components';
 
 import type { VisaDetail_data as VisaDetailType } from './__generated__/VisaDetail_data.graphql';
 
 type Props = {|
   +data: ?VisaDetailType,
 |};
 
-type VisaRequiredType = {
+type VisaRequiredType = {|
   +visaRequired: ?boolean,
-};
+|};
 
 const isVisaRequired = (
   passengers: $ReadOnlyArray<?VisaRequiredType>,
 ): boolean => passengers.some(passenger => passenger?.visaRequired);
 
-const VisaDetail = (props: Props) => {
+const Visa = (props: Props) => {
   const passengers = props.data?.passengers;
   const visaRequired = passengers && isVisaRequired(passengers);
```

Comment:
I know this is from a previous PR, but I think we should rather handle the case where passengers is null/undefined in `isVisaRequired`

```
const isVisaRequired = (
  passengers: ?$ReadOnlyArray<?VisaRequiredType>,
): boolean => {
if (passengers == null) return false;
return passengers.some(passenger => passenger?.visaRequired);	): boolean => passengers.some(passenger => passenger?.visaRequired);
}
```

then just call it like `isVisaRequired(props.data?.passengers)`

Reply:
Changes, looks better now